helium compiled in cortex-m0 targets

### LVGL version

9.4

### Platform

Linux, Raspberry Pi Pico target, CMake build system


### What happened?

Hi,

How it is supposed to exclude some assembly files or "components" when including LVGL as a CMake submodule? I'm using it on a Raspberry Pico (cortex-m0) which does not seem to support some instructionsn in lv_blend_helium.S :

```
[build] [14/550   0% :: 0.048] Building ASM object lvgl/CMakeFiles/lvgl.dir/src/draw/sw/blend/helium/lv_blend_helium.S.o
```

Reading the CMakeList the glob includes all the assembly files regardless the platform:

```
file(GLOB_RECURSE SOURCES ${LVGL_ROOT_DIR}/src/*.c
                          ${LVGL_ROOT_DIR}/src/*.cpp
                          ${LVGL_ROOT_DIR}/src/*.S)
```

At least maybe it would be nice to split the GLOB in sub components so that they can be excluded from the parent CMake file?


### How to reproduce?

```cmake 
# ------ LVGL
    set(LV_BUILD_CONF_PATH "${CMAKE_CURRENT_LIST_DIR}/system/lv_conf.h" CACHE PATH "LVGL config file" FORCE)

    set(LV_USE_DRAW_ARM2D_HELIUM OFF CACHE BOOL "" FORCE)
    # Dans les versions récentes de LVGL, il peut aussi être utile de forcer ceci :
    add_definitions(-DLV_USE_ARM2D_HELIUM=0)

    # Désactiver démos / examples / ThorVG
    set(CONFIG_LV_BUILD_DEMOS         OFF CACHE BOOL "" FORCE)
    set(CONFIG_LV_BUILD_EXAMPLES      OFF CACHE BOOL "" FORCE)
    set(CONFIG_LV_USE_THORVG_INTERNAL OFF CACHE BOOL "" FORCE)
```

Sure, here is the first one (so many more, same kind of error).
```
[build] [2/6  16% :: 0.047] Building ASM object firmware/lvgl/CMakeFiles/lvgl.dir/src/draw/sw/blend/helium/lv_blend_helium.S.o
[build] FAILED: firmware/lvgl/CMakeFiles/lvgl.dir/src/draw/sw/blend/helium/lv_blend_helium.S.o 
[build] /home/anthony/.pico-sdk/toolchain/14_2_Rel1/bin/arm-none-eabi-gcc -DLV_CONF_INCLUDE_SIMPLE -DLV_CONF_PATH=\"/mnt/work/git.codeberg/open-story-teller/firmware/system/lv_conf.h\" -DLV_KCONFIG_IGNORE -DLV_LVGL_H_INCLUDE_SIMPLE -DOST_USE_FF_LIBRARY -I/mnt/work/git.codeberg/open-story-teller/firmware/system/ff -I/mnt/work/git.codeberg/open-story-teller/core/chip32 -isystem /mnt/work/git.codeberg/open-story-teller/firmware/system -isystem /mnt/work/git.codeberg/open-story-teller/firmware/lvgl -isystem /mnt/work/git.codeberg/open-story-teller/build/firmware/lvgl -mcpu=cortex-m0plus -mthumb -Og -g -MD -MT firmware/lvgl/CMakeFiles/lvgl.dir/src/draw/sw/blend/helium/lv_blend_helium.S.o -MF firmware/lvgl/CMakeFiles/lvgl.dir/src/draw/sw/blend/helium/lv_blend_helium.S.o.d -o firmware/lvgl/CMakeFiles/lvgl.dir/src/draw/sw/blend/helium/lv_blend_helium.S.o   -c /mnt/work/git.codeberg/open-story-teller/firmware/lvgl/src/draw/sw/blend/helium/lv_blend_helium.S
[build] /home/anthony/.pico-sdk/toolchain/14_2_Rel1/arm-none-eabi/include/machine/_default_types.h: Assembler messages:
[build] /home/anthony/.pico-sdk/toolchain/14_2_Rel1/arm-none-eabi/include/machine/_default_types.h:41: Error: bad instruction `typedef signed char __int8_t'
```

--- a/src/draw/sw/blend/helium/lv_blend_helium.S
+++ b/src/draw/sw/blend/helium/lv_blend_helium.S
@@ -7,7 +7,7 @@
 #define __ASSEMBLY__
 #endif
 
-#include "lv_blend_helium.h"
+#include "../../../../lv_conf_internal.h"
 
 /*GCC Workaround: missing .note.GNU-stack section implies executable stack*/
 #ifdef __ELF__

